includes/polling/applications/bind.inc.php

```diff
@@ -127,7 +127,7 @@
 ];
 $metrics['incoming'] = $fields;
 
-$tags = ['name' => $name, 'app_id' => $app_id, 'rrd_def' => $rrd_def, 'rrd_name' => $rrd_name];
+$tags = ['name' => $name, 'app_id' => $app_id, 'client' => 'incoming_extended', 'rrd_def' => $rrd_def, 'rrd_name' => $rrd_name];
```

Comment:
client > type ?

Reply:
@mpikzink for some reason the client queries are split into two data groups...  Take a look at the keys in the two, there is no collision.

Reply:
OH, lol, I see i now, thanks.

Reply:
It is odd that incoming queries are split into two groups, but outgoing are not. I don't think there's really a good reason to tag them differently in TSDBs, other than to make them match the RRD (?)